From 5ef2e13774a87cd18e233ad4349b411e73798b26 Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <nathan@kernel.org>
Date: Thu, 24 Feb 2022 14:22:21 -0700
Subject: [PATCH 20/26] HACK: arm64: defconfig: Remove duplicate
 CONFIG_CAN_FLEXCAN=m

arch/arm64/configs/defconfig:184:warning: override: reassigning to symbol CAN_FLEXCAN

Signed-off-by: Nathan Chancellor <nathan@kernel.org>
---
 arch/arm64/configs/defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 846f9bdd680c..3baa33977ac7 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -181,7 +181,6 @@ CONFIG_CAN=m
 CONFIG_CAN_FLEXCAN=m
 CONFIG_CAN_RCAR=m
 CONFIG_CAN_RCAR_CANFD=m
-CONFIG_CAN_FLEXCAN=m
 CONFIG_CAN_MCP251XFD=m
 CONFIG_BT=m
 CONFIG_BT_HIDP=m
-- 
2.35.1

